WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/BlockEditorPage.java

```diff
@@ -74,10 +75,25 @@ public BlockEditorPage addPostSettings(String categoryName, String tagName) {
         return this;
     }
 
-    public void openPostSetting() {
+    /**
+     * Taps the three vertical dots menu button located at the editor screen top right.
+     * Since the tap does not always succeed on FTL, one retry is used
+     */
+    public void openPostKebabMenu() {
```

Comment:
I extracted the tap on 3-dots button from `openPostSetting` solely to justify having a function with [kebab](https://i.stack.imgur.com/OsXnO.png) in the title.